spec/models/comment_spec.rb

```diff
@@ -0,0 +1,6 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  it { is_expected.to belong_to(:task) }
+  it { is_expected.to validate_presence_of :text }
```

Comment:
Follow your code style
```suggestion
  it { is_expected.to validate_presence_of(:text) }
```